json_serializable/test/kitchen_sink_test.dart

```diff
@@ -6,102 +6,84 @@ import 'package:test/test.dart';
 
 import 'package:json_serializable/src/utils.dart';
 
-import 'test_files/kitchen_sink.dart';
+import 'test_files/kitchen_sink.dart' as nullable;
```

Comment:
nit, this prefix is a bit inconsistent (not abbreviated). Maybe just opt for no prefix? Or come up with a prefix and add it to `wrapped` as well.